docker/.env

```diff
@@ -0,0 +1,10 @@
+DTAAS_DIR='/home/Desktop/DTaaS'
+SERVER_DNS='foo.com'
+OAUTH_URL='https://gitlab.foo.com'
+CLIENT_ID='xx'
+CLIENT_SECRET='xx'
+OAUTH_SECRET='random-secret-string'
+ENV_JS_FILEPATH='/home/Desktop/DTaaS/deploy/config/client/env.local.js'
+OAUTH_CONF_FILEPATH='/home/Desktop/DTaaS/docker/conf.auth'
```

Comment:
please see the comments in the compose files. This variable might not be needed.